src/echo/freedom-module.ts

```diff
@@ -2,6 +2,8 @@
 /// <reference path='../logging/logging.d.ts' />
 /// <reference path='../freedom/typings/freedom.d.ts' />
 
+freedom['loggingprovider']().setConsoleFilter(['*:D']);
```

Comment:
this is default behavior per https://github.com/uProxy/uproxy-lib/blob/dev/src/loggingprovider/loggingprovider.ts#L33  Why needed?